pages/common/cf.md

```diff
@@ -0,0 +1,32 @@
+# cf
+
+> Command line tool to manage apps and services on Cloud Foundry.
+> More information: <https://docs.cloudfoundry.org/>.
+
+- Push an app using the default settings:
+
+`cf push {{app_name}}`
+
+- View the services available from your organization:
+
+`cf marketplace`
+
+- Create a service instance:
+
+`cf create-service {{service}} {{plan}} {{chosen_service_name}}`
+
+- Connect an application to a service:
+
+`cf bind-service {{app_name}} {{chosen_service_name}}`
+
+- Run a script whose code is included in the app, but runs independently:
+
+`cf run-task {{app_name}} {{script_command}} --name {{task_name}}`
```

Comment:
Could we enclose {{script_command}} in quotes please?

```suggestion
`cf run-task {{app_name}} "{{script_command}}" --name {{task_name}}`
```

Passing code as a CLI argument may potentially include special characters and more than 1 word, which we want to ensure get passed correctly.